channel/channel.go

```diff
@@ -13,22 +13,26 @@ import (
 	"github.com/statechannels/go-nitro/types"
 )
 
+type OnChainData struct {
+	LatestBlockNumber uint64 // the latest block number we've seen
+	Holdings          types.Funds
+	Outcome           outcome.Exit
+	StateHash         common.Hash
+}
+
+type OffChainData struct {
```

Comment:
This might be a preference thing but I think using the term `OffChain` in the context might be confusing. Generally I'd refer to any data not on-chain as `Offchain`. However in the context of this `Channel` struct `OffChain` is just the collection of supported states (some which could even be published on-chain via a challenge/response).

Reply:
More precisely it's a collection of signed states. If one of those supported states were published on chain via `challenge`, then `OnChain.StateHash` and `OnChain.Outcome` will be updated to reflect that.